llvm/lib/Target/NVPTX/NVPTXISelLowering.cpp

```diff
@@ -1134,7 +1143,8 @@ SDValue NVPTXTargetLowering::getSqrtEstimate(SDValue Operand, SelectionDAG &DAG,
                                              bool &UseOneConst,
                                              bool Reciprocal) const {
   if (!(Enabled == ReciprocalEstimate::Enabled ||
-        (Enabled == ReciprocalEstimate::Unspecified && !usePrecSqrtF32())))
+        (Enabled == ReciprocalEstimate::Unspecified &&
+         !usePrecSqrtF32(DAG.getMachineFunction()))))
```

Comment:
This condition is a bit of a pain to read.

AFAICT, it all boils down to just:
```
if (Enabled == ReciprocalEstimate::Disabled || usePrecSqrtF32(DAG.getMachineFunction())
```

Reply:
Shoot, sorry I missed this one!